search.go

```diff
@@ -64,13 +64,26 @@ func (req *SearchRequest) Size(size uint64) *SearchRequest {
 }
 
 // Sort sets how the results should be sorted.
-func (req *SearchRequest) Sort(name string, order Order) *SearchRequest {
+func (req *SearchRequest) Sort(params SortParams) *SearchRequest {
```

Comment:
struct bna do sort ka and add fields there. Map function me map wale kaam krte h